extensions/panache/rest-data-panache/runtime/src/main/java/io/quarkus/rest/data/panache/OperationProperties.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.rest.data.panache;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RUNTIME)
+@Target({ METHOD })
+public @interface OperationProperties {
```

Comment:
Could probably do with javadoc to mention this annotation is not mandatory unless you want to override the defaults.

Reply:
Done